src/com/carolinarollergirls/scoreboard/Main.java

```diff
@@ -52,11 +52,6 @@ private void parseArgv(String[] argv) {
             else if(arg.equals("--nogui") || arg.equals("-G")) {
                 gui = false;
             }
-
-            else if(arg.startsWith("--port=") || arg.startsWith("-p=")) {
-                String port = arg.split("=")[1];
-                ScoreBoardManager.setPropertyOverride("com.carolinarollergirls.scoreboard.jetty.JettyServletScoreBoardController.port", port);
```

Comment:
Why remove this? `-p=` is a lot easier to remember and type than `-Dcom.carolinarollergirls.scoreboard.jetty.JettyServletScoreBoardController.port=`. I'd rather think about adding something like `-l` and `--localhost` for that property.

Reply:
It's a fairly advanced feature that few users need (or are even aware exists), plus two ways to configure things is asking for confusion about which takes precedence.

Reply:
"Few users need it, so we can make it significantly more obscure" is not a convincing argument. And there's no need to even document the `-D` way - we can just treat it as something that happens to have that effect but is not an official feature at which point everybody who is able to figure out that it exists should also be able to figure out that `-p`/`-l` takes precedence (or that they should just leave it alone).

Reply:
Updated to take a different approach.